syne_tune/optimizer/schedulers/searchers/utils/__init__.py

```diff
@@ -10,3 +10,18 @@
 # on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 # express or implied. See the License for the specific language governing
 # permissions and limitations under the License.
+from syne_tune.optimizer.schedulers.searchers.utils.hp_ranges import (  # noqa: F401
+    HyperparameterRanges,
+)
+from syne_tune.optimizer.schedulers.searchers.utils.hp_ranges_factory import (  # noqa: F401
+    make_hyperparameter_ranges,
+)
+from syne_tune.optimizer.schedulers.searchers.utils.hp_ranges_impl import (  # noqa: F401
+    HyperparameterRangesImpl,
+)
+from syne_tune.optimizer.schedulers.searchers.utils.scaling import (  # noqa: F401
+    LinearScaling,
+    LogScaling,
+    ReverseLogScaling,
+    get_scaling,
+)
```

Comment:
I understand that those are all classes we want to make visible to users of syne-tune. All other objects in the uils module are only useful for developent, right?